File: spec/factories/inns.rb
# == Schema Information
#
# Table name: inns
#
#  id          :integer          not null, primary key
#  name        :string(255)
#  description :string(255)
#  rails       :integer
#  created_at  :datetime         not null
#  updated_at  :datetime         not null
#

FactoryGirl.define do
  factory :inn do
    name "MyString"
description ""
rails 1
  end

end

File: packages/db/src/schema/inns.ts
import {
  pgTable,
  serial,
  integer,
  text,
  timestamp,
} from "drizzle-orm/pg-core";

export const inns = pgTable("inns", {
  id: serial("id").primaryKey(),
  name: text("name").notNull(),
  description: text("description"),
  rails: integer("rails").notNull(),
  createdAt: timestamp("created_at").notNull().defaultNow(),
  updatedAt: timestamp("updated_at").notNull().defaultNow(),
});
